docs/content/1.get-started/2.configuration.md

```diff
@@ -45,6 +45,12 @@ export default defineNuxtConfig({
 })
 ```
 
+#### `weights`
```

Comment:
This was missing prior. _Technically_ this is a somewhat unrelated change to the current PR, but figured I'd save your inbox from yet another issue/PR combo for this inoffensive tweak 😁